src/Jobs/ProcessNodeChannelAcceptorJob.cs

```diff
@@ -227,7 +227,7 @@ await foreach (var response in resultAcceptor.ResponseStream.ReadAllAsync())
         catch (Exception e)
         {
             _logger.LogError(e, "Error on {JobName}", nameof(ProcessNodeChannelAcceptorJob));
-            throw new JobExecutionException(e, true);
+            throw new JobExecutionException(e, false);
```

Comment:
I modified this cronjobs to not refire immediately after an error, but to retry on the next interval. Let me know if this is ok.

Reply:
I would say so, this is what hangfire usually does